itests/gateway_test.go

```diff
@@ -260,19 +293,48 @@ func startNodes(
 
 	// Create a gateway client API that connects to the gateway server
 	var gapi api.Gateway
-	gapi, closer, err = client.NewGatewayRPCV1(ctx, "ws://"+srv.Listener.Addr().String()+"/rpc/v1", nil)
+	var _closer jsonrpc.ClientCloser
```

Comment:
Subjective: The `_` in name conventionally makes me think this should not be set explicitly (a style used in named return types). I would rename this to something else if `_` in name is there just to avoid clashing with the wrapper.